pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -91,37 +95,39 @@ type priorityWeight struct {
 //
 // User should specify priority weights in the config in this format:
 //
-//  actions: "reclaim, allocate, backfill, preempt"
-//  tiers:
-//  - plugins:
-//    - name: priority
-//    - name: gang
-//    - name: conformance
-//  - plugins:
-//    - name: drf
-//    - name: predicates
-//    - name: proportion
-//    - name: nodeorder
-//      arguments:
-//        leastrequested.weight: 1
-//        mostrequested.weight: 0
-//        nodeaffinity.weight: 1
-//        podaffinity.weight: 1
-//        balancedresource.weight: 1
-//        tainttoleration.weight: 1
-//        imagelocality.weight: 1
+//	actions: "reclaim, allocate, backfill, preempt"
+//	tiers:
+//	- plugins:
+//	  - name: priority
+//	  - name: gang
+//	  - name: conformance
+//	- plugins:
+//	  - name: drf
+//	  - name: predicates
+//	  - name: proportion
+//	  - name: nodeorder
+//	    arguments:
+//	      leastrequested.weight: 1
+//	      mostrequested.weight: 0
+//	      nodeaffinity.weight: 1
+//	      podaffinity.weight: 1
+//	      balancedresource.weight: 1
+//	      tainttoleration.weight: 1
+//	      imagelocality.weight: 1
+//	      podtopologyspread.weight: 1
 func calculateWeight(args framework.Arguments) priorityWeight {
 	// Initial values for weights.
 	// By default, for backward compatibility and for reasonable scores,
 	// least requested priority is enabled and most requested priority is disabled.
 	weight := priorityWeight{
-		leastReqWeight:         1,
-		mostReqWeight:          0,
-		nodeAffinityWeight:     1,
-		podAffinityWeight:      1,
-		balancedResourceWeight: 1,
-		taintTolerationWeight:  1,
-		imageLocalityWeight:    1,
+		leastReqWeight:          1,
+		mostReqWeight:           0,
+		nodeAffinityWeight:      1,
+		podAffinityWeight:       1,
+		balancedResourceWeight:  1,
+		taintTolerationWeight:   1,
+		imageLocalityWeight:     1,
+		podTopologySpreadWeight: 1,
```

Comment:
what's the default weight of podTopologySpread in kube-scheduler?

Reply:
If not set, it will be 1.